src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -200,17 +205,47 @@ export class GraphStyles {
     const isOutside = node.isOutside;
 
     let icons = '';
+    // Put a little more space between icons when a badge has multiple icons
+    const iconMargin = () => (icons === '' ? style(defaultIconMargin) : style({ marginRight: '2px' }));
     if (node.isRoot) {
-      icons = `<span class="${NodeIconRoot} ${iconMargin}"></span> ${icons}`;
+      icons = `<span class="${NodeIconRoot} ${iconMargin()}"></span> ${icons}`;
```

Comment:
I realize it was like this prior to the PR, but I think for trafficSource nodes (aka "root" nodes), the root icon should always be the leftmost icon.  Looking here, I think it ends up being the rightmost icon.  Let's make that change.